stuffioAPI/stuffioAPI/ViewModels/UserViewModel.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace stuffioAPI.ViewModels
+{
+    public class UserViewModel
+    {
+        public int Id { get; set; }
+
+        public string Name { get; set; }
+
+        public string Surname { get; set; }
+
+        public string Image { get; set; }
+
+        public string Email { get; set; }
+
+        public string Password { get; set; }
+    }
+}

```

Comment:
Inconsistent, what is required here?

Reply:
That is just a dummy field for Joseph that he requested to test the API calls in the app